### DISCUSSION TITLE
Connection lost, offline loop

### DISCUSSION THREAD

**Discussion by paulefde** [AntlerVC/firetable#1278] [Support / Q&A] [answered]
Hello, 

Rowy is losing the connection to firebase, it seems to be in a loop. What can cause this issue? 

https://github.com/rowyio/rowy/assets/96086864/503cb977-275d-4814-9e64-12d24e3fad6d

**Comment by BeeBombshell** [comment]
Hi @paulefde, thanks for sharing - we have been trying to debug this and replicate the issue but has been tricky to replicate. Any chance you can share a bit more information that can help us further to get to the bottom of this? 
- Do you have multiple tabs open with Rowy?
- Does the issue continue to happen if you clear your cache and login to Rowy again?
- If so are you able to use https://www.jam.dev Chrome extension to do a quick recording, as that will help us debug this issue better.

Thanks for your patience.

**Reply by paulefde** [reply]
I tested it with a new session on chrome. I normally work in Edge so I never logged in to rowy before. The problem is still there. 

This is the link where you can see how annoying it is especially when I want to import data. 

https://jam.dev/c/41cf509b-8a92-4cf6-9ebe-daa08d24767d

You can contact me @ paul@redframe.nl and you are welcome to look into my account. I really need this bug to get fixed as i am building out my database. When i'm done I need more projects so I will be paying for the higher plan. 

Regards, 

Paul

**Reply by paulefde** [reply]
I deleted my account, made a new account, with a new firebase project, followed the steps, checked on mobile, firefox, chrome, edge, incognito mode and I still get a connection lost error in a loop.

**Comment by BeeBombshell** [comment, marked as answer]
Hey @paulefde, we appreciate you bringing this issue to our attention. After a thorough investigation, we have identified the cause to be a third-party package that is no longer receiving active maintenance. Our dedicated team is already hard at work, actively exploring solutions to address this issue and find a suitable workaround. Thank you for your patience and understanding!

**Reply by paulefde** [reply]
Thank you! Hope the issue will resolve soon.

**Reply by BeeBombshell** [reply]
Hi @paulefde, **QUICK UPDATE!** This issue has now been fixed via the PR: https://github.com/rowyio/rowy/pull/1285.
